src/app/store.ts

```diff
@@ -11,14 +11,47 @@ import {
   UrlInputState,
 } from "../features/urlInput";
 import { TabsActions, tabsReducer, TabsState } from "../features/tabs";
-
 export type ApplicationState = {
   knownTags: KnownTagsState;
   items: ItemsState;
   urlInput: UrlInputState;
   tabs: TabsState;
 };
-export default configureStore<
+
+const localStorages = ["knownTags", "items", "tabs"];
+
+function saveToLocalStorage(state: ApplicationState) {
+  try {
+    localStorages.map((storage) => {
+      const storePart = storage as keyof typeof state;
```

Comment:
Never saw a thing like `as keyof typeof`. I guess it works. Do we need it tho?

Reply:
It seems like the only way to use "knownTags"/"items"/"tabs" string from external `localStorages` array as index of Redux store. When ommited, the error is: "Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'ApplicationState'."

But maybe I am missing something.